### ISSUE TITLE
[sub-issue] Contrast problem

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#324]
![image](https://cloud.githubusercontent.com/assets/7510842/5357700/064711ea-7fab-11e4-9eb1-8de007f0916d.png)

The contrast of text to background is insufficient.

==> Use at least #707070 (this Color is not in Webguidelines; Color in Webguidelines would be #454545 - but is quite dark)

**Comment by tonifisler** [comment]
Switched color to `#757575`.
